pytest_django/plugin.py

```diff
@@ -40,10 +39,96 @@
+             'assertRedirects', 'assertTemplateNotUsed', 'assertTemplateUsed',
+             'assertXMLEqual', 'assertXMLNotEqual'),
+    # from https://github.com/django/django/blob/stable/1.7.x/django/test/testcases.py#L162
+    #  and https://github.com/django/django/blob/stable/1.7.x/django/test/testcases.py#L722
+    (1, 7): ('assertContains', 'assertFieldOutput', 'assertFormError',
+             'assertFormsetError', 'assertHTMLEqual', 'assertHTMLNotEqual',
+             'assertInHTML', 'assertJSONEqual', 'assertNotContains',
+             'assertNumQueries', 'assertQuerysetEqual', 'assertRaisesMessage',
+             'assertRedirects', 'assertTemplateNotUsed', 'assertTemplateUsed',
+             'assertXMLEqual', 'assertXMLNotEqual'),
+    # from https://github.com/django/django/blob/stable/1.8.x/django/test/testcases.py#L142
+    #  and https://github.com/django/django/blob/stable/1.8.x/django/test/testcases.py#L751
+    (1, 8): ('assertContains', 'assertFieldOutput', 'assertFormError',
+             'assertFormsetError', 'assertHTMLEqual', 'assertHTMLNotEqual',
+             'assertInHTML', 'assertJSONEqual', 'assertJSONNotEqual',
+             'assertNotContains', 'assertNumQueries', 'assertQuerysetEqual',
+             'assertRaisesMessage', 'assertRedirects', 'assertTemplateNotUsed',
+             'assertTemplateUsed', 'assertXMLEqual', 'assertXMLNotEqual'),
+    # from https://github.com/django/django/blob/stable/1.9.x/django/test/testcases.py#L158
+    #  and https://github.com/django/django/blob/stable/1.9.x/django/test/testcases.py#L818
+    (1, 9): ('assertContains', 'assertFieldOutput', 'assertFormError',
+             'assertFormsetError', 'assertHTMLEqual', 'assertHTMLNotEqual',
+             'assertInHTML', 'assertJSONEqual', 'assertJSONNotEqual',
+             'assertNotContains', 'assertNumQueries', 'assertQuerysetEqual',
+             'assertRaisesMessage', 'assertRedirects', 'assertTemplateNotUsed',
+             'assertTemplateUsed', 'assertXMLEqual', 'assertXMLNotEqual'),
+    # from https://github.com/django/django/blob/stable/1.10.x/django/test/testcases.py#L155
+    #  and https://github.com/django/django/blob/stable/1.10.x/django/test/testcases.py#L797
+    (1, 10): ('assertContains', 'assertFieldOutput', 'assertFormError',
+              'assertFormsetError', 'assertHTMLEqual', 'assertHTMLNotEqual',
+              'assertInHTML', 'assertJSONEqual', 'assertJSONNotEqual',
+              'assertNotContains', 'assertNumQueries', 'assertQuerysetEqual',
+              'assertRaisesMessage', 'assertRedirects', 'assertTemplateNotUsed',
+              'assertTemplateUsed', 'assertXMLEqual', 'assertXMLNotEqual'),
+}
 
 # ############### pytest hooks ################
 
+def populate_namespace():
+    def _wrapper(name):
+
+        def assertion_func(*args, **kwargs):
+            from django.test import TestCase as DjangoTestCase
+
+            getattr(DjangoTestCase('run'), name)(*args, **kwargs)
```

Comment:
What does `('run')` do in this case? Just trying to understand why it's necessary